arcgis-ios-sdk-samples/Layers/Add ENC exchange set/AddENCExchangeSetViewController.swift

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class AddENCExchangeSetViewController: UIViewController {
+    /// The map view managed by the view controller.
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap(basemap: .oceans())
+        }
+    }
+    
+    /// Load the ENC dataset and add to the map view.
+    ///
+    /// - Parameter mapView: The map view managed by the view controller.
+    func addENCExchangeSet(mapView: AGSMapView) {
+        guard let map = mapView.map else { return }
+        // Load catalog file in ENC exchange set from bundle.
+        let catalogURL = Bundle.main.url(
+            forResource: "CATALOG",
+            withExtension: "031",
+            subdirectory: "ExchangeSetwithoutUpdates/ExchangeSetwithoutUpdates/ENC_ROOT"
+        )!
+        let encExchangeSet = AGSENCExchangeSet(fileURLs: [catalogURL])
+        
+        // URL to the "hydrography" data folder that contains the "S57DataDictionary.xml" file.
+        let hydrographyDirectory = Bundle.main.url(
+            forResource: "S57DataDictionary",
+            withExtension: "xml",
+            subdirectory: "hydrography/hydrography"
+        )!
+        .deletingLastPathComponent()
+        // Set environment settings for loading the dataset.
+        let environmentSettings = AGSENCEnvironmentSettings.shared()
+        environmentSettings.resourceDirectory = hydrographyDirectory
+        // The SENC data directory is for temporarily storing generated files.
+        environmentSettings.sencDataDirectory = FileManager.default.temporaryDirectory
+        
+        encExchangeSet.load { [weak self] error in
+            guard let self = self else { return }
+            if let error = error {
+                self.presentAlert(error: error)
+            } else {
+                var ENCLayers = [AGSENCLayer]()
```

Comment:
FYI: In Swift, variable names almost always begin with a lowercase letter. When the name begins with an acronym, the convention Apple uses is to lowercase all the letters in the acronym (like [`UIColor.cgColor`](https://developer.apple.com/documentation/uikit/uicolor/1621954-cgcolor), for example). So a better name for this would be `encLayers`. By the way, this naming convention differs from Objective-C.